cmd/kusk/cmd/kubectl.go

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
Please re-run:

```
$ kusk docs
```

So that it re-generates the docs and ensure the `install` command is no longer present, and that `kubectl` is documented and present.

Reply:
we are not doing it anymore